protocol/message_persistence.go

```diff
@@ -116,6 +118,8 @@ func (db sqlitePersistence) tableUserMessagesAllFieldsJoin() string {
 		m1.image_payload,
 		m1.image_type,
 		COALESCE(m1.album_id, ""),
+		COALESCE(m1.image_width, ""),
+		COALESCE(m1.image_height, ""),
```

Comment:
I think this needs to be coalesced to `0` since it's `INTEGER`

Reply:
That's right! Fixed it.